### DISCUSSION TITLE
Segunda Oportunidad - ONBOARDING

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#192] [Q&A]
Mission Comander @romarpla buenas noches, fíjese que me llego este correo, la verdad no se si preocuparme o todo esta bien.

![image](https://user-images.githubusercontent.com/99059639/156274452-9b09f0eb-6d26-4df2-9740-822fbf5e8a84.png)

De hecho ya estoy en el Frontend, porque al enviar mis katas del Onboarding me llegó automáticamente el Link al Frontend.

¿A caso será solo un mensaje automatizado?

**Comment by CrispisMAB** [comment]
Igual tengo la misma duda y pensaba mandarlas de nuevo, puesto pensé que no habían recibido mis katas :(((

**Reply by CesarMezaB** [reply]
También me llego ese correo

**Comment by JuanSalvador73** [comment]
Hay que escribirles al correo de Innovación Virtual que nos dieron desde el comienzo. Personalmente creo que es un error, pero mas vale asegurarse.

Saludos.
